### PR TITLE
Change backquote to singlequote in messages

### PR DESCRIPTION
**Pull request by znz**
Because Ruby 3.4 changed.

See https://bugs.ruby-lang.org/issues/16495

### CODE REVIEW

File: lib/webrick/cgi.rb [hunk 1]
```diff
@@ -160,7 +160,7 @@ def service(req, res)
         __send__(method_name, req, res)
       else
         raise HTTPStatus::MethodNotAllowed,
-              "unsupported method `#{req.request_method}'."
+              "unsupported method '#{req.request_method}'."
```
Comment (ioquatix) [inline lib/webrick/cgi.rb:163]: I think it's even better (safer) to use `req.request_method.inspect` here and in other places. What do you think?

**Review by ioquatix — CHANGES_REQUESTED** [review]

File: lib/webrick/cgi.rb (hunk 1 above)
Comment (znz) [reply lib/webrick/cgi.rb:163]: I agree that it is safer.
I think it would be better to do it in a separate pull request.

Comment (ioquatix) [reply lib/webrick/cgi.rb:163]: Okay, do you want to open a new pull request?

Comment (znz) [reply lib/webrick/cgi.rb:163]: I don't have time to do it, so I'll leave it to you.

**Pull request merged**